examples/postInit/pyrotech.groovy

```diff
@@ -74,6 +74,11 @@ mods.pyrotech.campfire.recipeBuilder()
 
 mods.pyrotech.campfire.add('apple_to_dirt', item('minecraft:apple'), item('minecraft:dirt'), 1000)
 
+// Stone & Refactored Oven
```

Comment:
either:
- in the `gradle.properties` file, enable `debug_generate_examples` and `debug_generate_wiki`, launch, then check the `groovy.log` file for any errors.
- join the game, run `/gs generateExamples` and `/gs generateWiki`, then check the `groovy.log` file for any errors